src/SeaIceDeformation/M01_d03_compute_deformations.py

```diff
@@ -80,8 +80,8 @@ def compute_deformations():
     sTime, eTime, \
         sLat1, sLat2, sLat3, sLon1, sLon2, sLon3, \
         eLat1, eLat2, eLat3, eLon1, eLon2, eLon3, \
-        div, shear, vrt \
-        = ([] for i in range(17))
+        div, shear, vrt, idx1, idx2, idx3, no \
+        = ([] for i in range(21))
```

Comment:
would it be possible to avoid a hardcoded number here and use `len` or something like that?

Reply:
This is the most convenient way I (and clearly Beatrice) have found to initialize multiple lists with names that are not iterative integers. List comprehension gets very messy when dealing with custom names such as these.

Reply:
I see. I was just curious. I'll go ahead and approve.